sailor/assetcentral/system.py

```diff
@@ -60,7 +63,7 @@ def _prepare_components(self):
             self.components = EquipmentSet([])
 
     def get_indicator_data(self, start: Union[str, pd.Timestamp, datetime.timestamp, datetime.date],
-                           end: Union[str, pd.Timestamp, datetime.timestamp, datetime.date]) -> TimeseriesDataset:
+                           end: Union[str, pd.Timestamp, datetime.timestamp, datetime.date]) -> 'TimeseriesDataset':
```

Comment:
I think to be in line with the other modules I'd prefer to use `from __future__import annotations` and no quotes here as well